### PR TITLE
Fix navigation to calendar in create-screening

### PR DESCRIPTION
**Pull request by xmano**
Test is more reliable with this.

### CODE REVIEW

File: apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts [hunk 1]
```diff
@@ -46,27 +46,17 @@ describe('User create a screening', () => {
...
+    (new FestivalMarketplaceHomePage()).goToDashboard();
+    const homePage = new FestivalDashboardHomePage();
+    const eventPage: EventPage = homePage.goToCalendar();
+
+    [[0, tomorrow, false], [1, tomorrow, true], 
+      [0, twodayslater, true], [1, twodayslater, false]].forEach((x: any, index:number) => {
+        const [i, d, p] = x;
+        const eventName = EVENTS[i].event + index;
+        eventPage.createEvent(eventName, d, 
+                EVENTS[i].movie.title.international, p);
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts:59]: please put it in a single line or use correct indent (2 spaces)

File: apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts [hunk 2]
```diff
@@ -46,27 +46,17 @@ describe('User create a screening', () => {
...
-          eventPage.createEvent(eventName, d, 
-                  EVENTS[i].movie.title.international, p);
-      });
+    cy.log('From Marketplace Homepage, navigating to calendar...');
+    (new FestivalMarketplaceHomePage()).goToDashboard();
+    const homePage = new FestivalDashboardHomePage();
+    const eventPage: EventPage = homePage.goToCalendar();
+
+    [[0, tomorrow, false], [1, tomorrow, true], 
+      [0, twodayslater, true], [1, twodayslater, false]].forEach((x: any, index:number) => {
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts:55]: this [[][]] is hard to read

**Review by phpgeekfr — APPROVED** [review]

File: apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts (hunk 1 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts:59]: ok

File: apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts (hunk 2 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts]: refactored to array

**Pull request merged**